### DISCUSSION TITLE
hyper to hashbase

### DISCUSSION THREAD

**Discussion by SPierceMyers** [beakerbrowser/beaker#1906] [Q&A]
Hi all!

In Beaker 0.8 it was straightforwad for a non-coder (like me) to update my dat page, sync to hashbase, and redirect my domain name to hashbase. 

But today the hashbase archive upload was perpetually stuck at 0%, so i downloaded Beaker 1.1 in hopes that this solved the issue. It has a lot of features I don't understand, but my legacy dat website folder is stored locally. 

If anyone could point me to how I seed to hashbase with //hyper from Beaker 1.1 I would much appreciate it!

I'm also open to other ways to host that allow me others to link over from https. But perhaps Hashbase runs against the whole ethos of Beaker?

My Best,
Pierce

**Comment by pfrazee** [comment]
Yeah there's not a hashbase for hyper yet, but I'm working on a new kind of infra for servers using hyper with https://github.com/pfrazee/ctzn, which focuses on social networking atm but may expand into things like hyper-hosting so stay tuned

**Reply by skyfaller** [reply]
It looks like the ctzn repo says "[DEPRECATED]"? If this project is no longer active and will not get to hyper-hosting, is there another hyper-hosting project?

**Reply by pfrazee** [reply]
CTZN transformed into the Atek project, which is actually getting released tomorrow. It's a general-purpose cloud architecture for running hypercore (and other p2p/web3) applications

- https://github.com/atek-cloud
- https://twitter.com/atek_cloud
